lib/commands/kendoui-install.ts

```diff
@@ -0,0 +1,124 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import temp = require("temp");
+import * as path from "path";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUIInstallCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $fs: IFileSystem,
+		private $httpClient: Server.IHttpClient,
+		private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let packages = this.getKendoPackages().wait();
+
+			let downloadUri: string;
+			if(this.$options.latest) {
+				let latestPackage = _.first(packages);
+				let sameDateItems = _.filter(packages, pack => pack.Version === latestPackage.Version);
+				if(sameDateItems.length > 1) {
+					downloadUri = this.selectKendoVersion(sameDateItems).wait();
+				} else {
+					downloadUri = latestPackage.DownloadUrl;
+				}
+			} else {
+				downloadUri = this.selectKendoVersion(packages).wait();
+			}
+
+			let confirm = this.$options.force || this.$prompter.confirm(
+				"This operation will overwrite existing Kendo UI framework files and " +
+				"any changes will be lost. ".red.bold +
+				"Are you sure you want to continue?",
+				() => true).wait();
+			if (!confirm) {
+				return;
+			}
+
+			this.updateKendoFiles(downloadUri).wait();
+
+		}).future<void>()();
+	}
+
+	private selectKendoVersion(packages: Server.IKendoDownloadablePackageData[]): IFuture<string> {
+		return ((): string => {
+			this.$logger.out("You can download and install the following Kendo UI packages.");
+			this.$logger.out(this.getKendoPackagesAsTable(packages));
+			let schema: IPromptSchema = {
+				type: "input",
+				name: "packageIdx",
+				message: "Enter the index of the package that you want to install",
+				validate: (value: string) => {
+					let num = parseInt(value, 10);
+					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
```

Comment:
The length of the displayed packages can only be equal to 1 if `--latest` flag is set along with either `--professional` or `--core`. 
Should that be the case, this message is never shown in the first place